src/exception.cr

```diff
@@ -91,3 +91,16 @@ class DivisionByZero < Exception
     super(message)
   end
 end
+
+# Raised when a feature is not implemented yet or when it cannot be implemented
+# on a certain system or platform
+#
+# ```
+# Sysctl.get_int("net.ipv4.forward") # Sysctl doesn't exist on Windows
```

Comment:
Typically we use `# raises ExceptionType (message)` when documenting raising in examples [(ref)](https://github.com/crystal-lang/crystal/blob/master/src/io/memory.cr#L380)